setup_support.py

```diff
@@ -361,3 +411,100 @@ def _provide_lean_script_contents(
         setup_kwargs['scripts'] = scripts
 
     return setup_kwargs
+
+
+def get_long_description_from_README():
+    """Read README.md, convert to .rst using pypandoc
+
+    If pypandoc is not available or fails - just output original .md.
+
+    Returns
+    -------
+    dict
+      with keys long_description and possibly long_description_content_type
+      for newer setuptools which support uploading of markdown as is.
+    """
+    # PyPI used to not render markdown. Workaround for a sane appearance
+    # https://github.com/pypa/pypi-legacy/issues/148#issuecomment-227757822
+    # is still in place for older setuptools
+
+    README = opj(dirname(__file__), 'README.md')
+
+    ret = {}
+    if LooseVersion(setuptools.__version__) > '38.6.0':
```

Comment:
Woops. Sorry about the sloppy suggestion.

```suggestion
    if LooseVersion(setuptools.__version__) >= '38.6.0':
```

Reply:
Thanks -- I just amended the last commit, will force push later